src/docs/product/alerts/create-alerts/issue-alert-config.mdx

```diff
@@ -16,14 +16,18 @@ Specify which <PlatformLink to="/configuration/environments/">environment(s)</Pl
 
 The “Environment” dropdown list here has the same environments that are available for the selected project in the common “Environment” filter dropdown (this does not include hidden environments). Selecting "All Environments" is equivalent to having no environment filter.
```

Comment:
This option, as well as others need to be moved on this page to match the order they now appear in the UI.

Reply:
:+1: